### ISSUE TITLE
Packer Registry (like the Terraform Registry)

### ISSUE THREAD

**Issue by pbar1** [hashicorp/packer#7011]
What if there was a Packer Registry, analogous to the Terraform Registry, where people could submit Packer build instructions, just like they already can with their Terraform modules? This would allow for a nice place to find things like custom AMIs (such as the [Amazon EKS-optimzed AMI](https://github.com/awslabs/amazon-eks-ami)).

**Comment by SwampDragons** [comment]
This is a cool idea. I'll ask around internally if it's something our product team would be interested in.

**Comment by ladar** [comment]
Are you aware of this page? 

https://www.packer.io/community-tools.html

Or my own repo, which has configs for 32 distros, and counting...

https://github.com/lavabit/robox/

**Comment by xka5h** [comment]
Something like this is coming in HCP soon - https://www.hashicorp.com/blog/announcing-hcp-packer

**Comment by SwampDragons** [comment]
@xka5h The HCP Packer registry isn't really analogous to the Terraform registry; it's a tool for tracking image artifacts rather than build instructions.

**Comment by xka5h** [comment]
@SwampDragons Yes, thanks for clarifying. I made a slight tweak to my comment to reflect that.

**Comment by freakinhippie** [comment]
@SwampDragons did anything ever come from this? I believe this would be a useful feature as well.
